docs/swagger.json

```diff
@@ -1,477 +1,692 @@
-              "description" : "Bad Request",
-              "schema" : {
-                "$ref" : "#/definitions/ErrorDetails"
-              }
-            },
-            "401" : {
-              "description" : "Unauthorized"
-            },
-            "500" : {
-              "description" : "Internal Server Error",
-              "schema" : {
-                "$ref" : "#/definitions/ErrorDetails"
-              }
-            },
-            "503" : {
-              "description" : "Service unavailable"
-            },
-            "504" : {
-              "description" : "Gateway Timeout"
-            }
+  "swagger" : "2.0",
+  "info" : {
+    "description" : "Vipps PSP API gives PSP ability to process Vipps payment as a part of their payment offerings. By using PSP API, PSP’s still have full transactional control for payments initiated in their payment terminals.\nVipps PSP API allows PSP’s to initiate payments for end users to accept with the Vipps mobile app. After customer confirms the payment,Vipps enrich payment data with encrypted credit card details and hand over payment processing to PSP",
+    "version" : "1.0",
+    "title" : "PSP API V2"
+  },
+  "basePath" : "/dwo-api-application",
+  "tags" : [ {
+    "name" : "dwo-merchant-query-controller",
+    "description" : "DWO Merchant Query Controller"
+  }, {
+    "name" : "psp-payment-controller",
+    "description" : "PSP Payment Controller"
+  }, {
+    "name" : "psp-initiate-payment-controller",
+    "description" : "PSP Initiate Payment Controller"
+  } ],
+  "paths" : {
+    "/v2/psppayments/init" : {
+      "post" : {
+        "tags" : [ "psp-initiate-payment-controller" ],
+        "summary" : "Initiate a PSP Payment",
+        "operationId" : "initiatePaymentV2UsingPOST",
+        "consumes" : [ "application/json;charset=UTF-8" ],
+        "produces" : [ "application/json;charset=UTF-8" ],
+        "parameters" : [ {
+          "name" : "Authorization",
+          "in" : "header",
+          "description" : "Bearer JWT-TOKEN",
+          "required" : true,
+          "type" : "string",
+          "default" : "Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJvcmdhbml6YXRpb25OdW1iZXIiOiI5MTg3MTM4NjciLCJpc3MiOiJ2aXBwcy5ubyIsImV4cCI6MTY0NDc0Mzg5NywiYWNjb3VudE51bWJlciI6IjE4MDAwMDAwMDAwIn0.mqO4RsidM_-twRfCvgNnXUUPrXaTgFVMvkJTVmJgtmbHziMVxHnG9moHpEj90_A3MRPp_jucdgMnqixgExOykJCLX6VsiUB0qMLXTgBE7Mmkz3lk0jCgv7OwYDJuLksrXvLepD68KdyDTCVwPxdgqh_H9Tea7S3gSYUnYoNkNP5h7Eyvai6U7BuJVcoE5ckO1ZjGmOvNlfHUbAZYBH8I5RV96bnmEtdtkojrkoZtT7HjueyTZODfdaOyvI5VfMLKbr9qGkYKP0UBKTp9ST5eV-W_Y062PgOsfRNXL2-ltKgn0vDkpUWYGiA8Y98ondXLF1kQIc8pAFPAwhzWG31oM_wKeOjjjK-2lMg1X1LjTk-zbq0HBilhQzbXdW0PHws0p1UZgviW-bdIwRfcjn1ysEqzS0HzDuaRTGIQDeld3MrtcwLLce7PT4rSGZeoJM-pUTidztzX48pFUixLm-aa1-SWnbRIqUufmsylSJNZXjgS06pISGUMKPiAdSjINY17x8YkeptS1B_r56oshRY8BFffCPxtMJI1BAWYQBCQ8WvUadlWNrQ477kUuvauEPRK_eyXT83cANaEa39lBPz4J3PHAE6grV4c3G_rClFzaFl1mzrhUABKJ3CpP_q7LmkboXSI5xx0BkyIb4wG_eG1s7_E_P_T0hIj3CTBNZnu7mI"
```

Comment:
Is this Bearer X.... a specific example? If its just a random value it will be confusing for a non technical reader.

Reply:
Changed the description.